crates/graphql/src/schema/query_root.rs

```diff
@@ -316,6 +346,41 @@ impl QueryRoot {
             .map_err(Into::into)
     }
 
+    fn featured_listings(
+        &self,
+        context: &AppContext,
+        #[graphql(description = "Limit for query")] limit: i32,
+        #[graphql(description = "Offset for query")] offset: i32,
+    ) -> FieldResult<Vec<ListingReceipt>> {
+        let conn = context.shared.db.get().context("failed to connect to db")?;
+
+        let holaplex_auction_house_address = "9SvsTjqk3YoicaYnC4VW1f8QAN9ku7QCCk6AyfUdzc9t";
+        // TODO expand with other wallets to exclude
+        let sellers_to_exclude = vec!["7r8oBPs3vNqgqEG8gnyPWUPgWuScxXyUxtmoLd1bg17F"];
```

Comment:
This needs to be fleshed out _before_ being merged, so that we dont just return a bunch of listings from holaplex employees.

@kespinola @kristianeboe

Reply:
This should be changed to load from the runtime environment before merging.  Can discuss ways to make that work.